renaming cluster-with-network-restrictions (#3545)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.4/clusters/attach-cluster/cluster-with-network-restrictions/index.md

---
layout: layout.pug
navigationTitle: Attach a cluster with networking restrictions
title: Attach a cluster with networking restrictions
menuWeight: 10
excerpt: How to attach an existing cluster that has additional networking restrictions
beta: true
---

Use this option when you want to attach a cluster that is in a DMZ, behind a proxy server or a firewall, or that requires additional access information. This procedure gathers the information required to create a kubeconfig file for the network tunnel between Kommander and the cluster you want to attach.

1. In the selected workspace Dashboard, select the **Add Cluster** option in the **Actions** dropdown menu at the top right.

> 1. On the Add Cluster page, select **Attach Cluster**.

Reviewer: ```suggestion
1. On the **Add Cluster** page, select **Attach Cluster**.
```

1. Select the **Cluster has networking restrictions** card to display the configuration page.

![Add Cluster Networking Options](/dkp/kommander/1.4/img/cluster-has-networking-restrictions.png)

1. Enter the **Cluster Name** of the cluster you're attaching and select a **Workspace** from the dropdown list (if entering the **Add Cluster** menu from the Global workspace).

> 1. Create one or more new Labels as needed.

Reviewer: ```suggestion
1. Create additional new Labels as needed.
```

1. Select the **Load Balancer Hostname** which is the Ingress for the cluster from the dropdown menu. You will want the hostname to match the Kommander Host cluster that you are attaching your existing cluster with network restrictions to. 

> 1. Specify the **URL Path Prefix** for your Load Balancer Hostname. This URL path that will serve as the prefix for the specific tunnel services you want to expose on the Kommander management cluster. If no value is specified, value will default to `/`.

Reviewer: ```suggestion
1. Specify the **URL Path Prefix** for your Load Balancer Hostname. This URL path will serve as the prefix for the specific tunnel services you want to expose on the Kommander management cluster. If no value is specified, value defaults to `/`.
```

1. (Optional) Enter a value for the **Hostname** field.

> 1. If you haven't attached this cluster before, you will want to create a new secret in the **Root CA Certificate** drop down menu. To do this, go to your Konvoy management cluster, and enter:

Reviewer: ```suggestion
1. If you have not attached this cluster before, you must create a new secret in the **Root CA Certificate** drop down menu. To do this, go to your Konvoy management cluster, and enter:
```

   ```shell
   hostname=$(kubectl get service -n kubeaddons traefik-kubeaddons -o jsonpath="{.status.loadBalancer.ingress[0].hostname}")
   b64ca_cert=$(kubectl get secret -n cert-manager kubernetes-root-ca -o=go-template='{{index .data "tls.crt"}}')
   ```

>    To view your base64 encoded Kubernetes secret value so you can copy it to your clipboard to paste into this field **Root CA Certificate** field, run:

Reviewer: ```suggestion
   To view your base64 encoded Kubernetes secret value to copy and paste into the **Root CA Certificate** field, run:
```

---
layout: layout.pug
navigationTitle: Attach a cluster with networking restrictions
title: Attach a cluster with networking restrictions
menuWeight: 10
excerpt: How to attach an existing cluster that has additional networking restrictions
beta: true
---

> Use this option when you want to attach a cluster that is in a DMZ, behind a proxy server or a firewall, or that requires additional access information. This procedure gathers the information required to create a kubeconfig file for the network tunnel between Kommander and the cluster you want to attach.

Reviewer 2: is it exactly the same file here ? Did you just move it ? This would help us reviewing it.

Author: @hectorj2f - same file, just some minor adjustments.

Had to rename the file so it was consistent, and so the link would work in the app.
